src/content/docs/new-relic-solutions/new-relic-one/workloads/use-workloads.mdx

```diff
@@ -185,9 +165,8 @@ You can view a topological map that displays how your workloads relate to and af
 
 To access workloads maps:
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then select **All capabilities**.
-2. Click the **Workloads** tile.
-3. In the top pane, click **Maps**. 
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Workloads**.
+2. Click **Map**.
```

Comment:
I can't see this option...I see a lot of tiles and 2 combo boxes, one for View and another for View mode.

Reply:
i realized i need to add 'select a workload'